groq-model-provider/main.py

```diff
@@ -0,0 +1,160 @@
+import base64
+import json
+import os
+import time
+from typing import AsyncIterable
+
+import requests
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse, StreamingResponse
+from openai import OpenAI
+
+debug = os.environ.get("DEBUG", False) == "true"
+uri = "http://127.0.0.1:" + os.environ.get("PORT", "8000")
+
+# Initialize Groq client
+client = OpenAI(
+    base_url="https://api.groq.com/openai/v1",
+    api_key=os.environ.get("ACORN_GROQ_MODEL_PROVIDER_API_KEY", ""),
```

Comment:
Not sure if this should follow the new branding name that starts with `OBOT`

Reply:
updated